website/content/docs/operations/key-management.mdx

```diff
@@ -0,0 +1,60 @@
+---
+layout: docs
+page_title: Key Management
+description: Learn about the key management in Nomad.
+---
+
+# Key Management
+
+Nomad servers maintain an encryption keyring used to encrypt [secure
+variables] and sign task [workload identities]. The servers store key
```

Comment:
elsewhere we use caps on Secure Variables, should it be that way here?

Reply:
Good catch